packages/design-system-scripts/gulp/build.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @file The build tasks handle compiling and optimizing both the
+ *  package assets and the documentation site. Essentially makes
+ *  everything production-ready.
+ */
+const babel = require('gulp-babel');
+const count = require('gulp-count');
+const del = require('del');
+const gulp = require('gulp');
+const getPackageName = require('./common/getPackageName');
```

Comment:
I _think_ we can get rid of this because our directories will be named after their npm packages right?

Reply:
although maybe we cant assume that for child DS, nevermind